pset-iterators.js

```diff
@@ -62,8 +62,13 @@ declared with const
 console.log('Problem 1:')
 
 // Add your code below this line
+const presidentsNamedJamesArr = presidents.filter(function (president){
```

Comment:
@kyleligman nice use of `.indexOf()`,  `.startsWith()` and `.substring()` could have also been used